kedro/framework/hooks/manager.py

```diff
@@ -89,13 +89,13 @@ def _register_hooks_setuptools(
             plugin_names.add(f"{dist.project_name}-{dist.version}")
 
     if disabled_plugin_names:
-        logging.info(
+        logging.getLogger(__name__).info(
```

Comment:
Optional: We can have this factored out on one line at the top.

Reply:
Good idea, done!